lib/spoom/cli.rb

```diff
@@ -14,6 +14,7 @@ class Main < Thor
       extend T::Sig
 
       class_option :no_color, desc: "Don't use colors", type: :boolean
+      map T.unsafe(%w[--version -v] => :__print_version)
```

Comment:
We map both `--version` and `-v` options to call the `__print_version` method.

We do that a bit differently because we want `-v` to actually behave like a Thor command: `spoom --version` should not be usable with another command. Like `spoom --version tc` makes no sense.

That way we stop the command dispatch.